### PR TITLE
Add/modify variable names related to surface height/pressure

### PR DESCRIPTION
**Pull request by svahl991**
This adds two names relating to height at surface, and modifies `surface_air_pressure` for the naming rule change coming in #72.

(I realize that there is more renaming of existing variables to be done as a product of #72, so I can remove `surface_air_pressure` from this PR if people feel it is better to do that rename in a group with the others. It's just that this name is of particular interest to the fast-approaching JEDI renaming code sprint, so I wanted to be sure to get it in.)

### CODE REVIEW

**Review by gold2718 — APPROVED** [review]
Looks good!

**Review by climbfuji — APPROVED** [review]

File: standard_names.xml [hunk 1]
```diff
@@ -243,6 +243,10 @@
                    long_name="geopotential height w.r.t. sea level">
       <type kind="kind_phys" units="m">real</type>
     </standard_name>
+    <standard_name name="geopotential_height_at_surface"
+                   long_name="geopotential height at local surface">
```
Comment (nusbaume) [inline standard_names.xml:247]: We might need to specify what this field is in relation to, for example:

```suggestion
                   long_name="geopotential height at local surface w.r.t. sea level">
```

**Review by nusbaume — CHANGES_REQUESTED** [review]
Looks good!  Just had one request.

File: standard_names.xml (hunk 1 above)
Comment (svahl991) [reply standard_names.xml]: I've now added `w.r.t. sea level`. Also capitalized `Geopotential` in `long_name` since that seems to be standard.

**Review by cacraigucar — APPROVED** [review]

**Review by ss421 — APPROVED** [review]

**Comment by climbfuji** [comment]
@mkavulich @dustinswales this is ready for your final review and merging - thanks!

**Review by dustinswales — APPROVED** [review]
I see no issue with these changes/additions.

**Pull request merged**